lib/obs_github_deployments/cli/commands/is_locked.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module ObsGithubDeployments
+  module CLI
+    module Commands
+      class IsLocked < Dry::CLI::Command
+        # desc "Checks if the deployment is locked or not"
+
+        argument :repository, required: true, desc: "The GitHub repository to check for locked"
+        argument :access_token, required: true, desc: "The required GitHub access token"
```

Comment:
We should align this with the `set-lock`, there we call it `token`. I used just `token` because its less to type. But we can also agree on something else, but should be consistent

Reply:
Let's use `token` :+1: